### DISCUSSION TITLE
future ready talent

### DISCUSSION THREAD

**Discussion by vb1311** [github/india#302] [GitHub Platform]
I have watched industry session videos many times but it is still buffering even though i am having a good internet connection and i am not getting the complete status and the date of completeing the industry sessions are approaching please help me solve the problem as soon as possible

**Comment by Daipayan-Das** [comment]
watch the videos in one row without any buffer or skip

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by FRTMentor** [comment]
Please reach out our support team.
support@futurereadytalent.in
